cartridges/int_adyen_overlay/cartridge/scripts/util/adyenConfigs.js

```diff
@@ -4,7 +4,7 @@ var dwsystem = require('dw/system');
 var adyenCurrentSite = dwsystem.Site.getCurrent();
 function getCustomPreference(field) {
   var customPreference = null;
-  if (adyenCurrentSite && adyenCurrentSite.getCustomPreferenceValue(field)) {
+  if (adyenCurrentSite) {
```

Comment:
So the problem was that if we had a `field` ( `customPref `) that is `bool` type (for example `AdyenOneClickEnabled`) and it is set to `false`, it will skip this if statement because the second condition is not fulfilled and `customPreference` will be left on `null` and not `false` as it should.
Also it doesn't make to much sense to have the second condition because if we the `filed` name (`customPref`) doesn't exist it will ret null and it will set the `customPreference` to `null` anyways